README.md

```diff
@@ -28,53 +28,84 @@ There are three different modes for the Moodle user to link files from oCIS to M
+      ```
+      moodle will now be available under http://localhost:8000 
+    - Other installation methods:
+        - [Install and run moodle](https://docs.moodle.org/402/en/Installing_Moodle)
+        - copy / clone the code of the repository into the `repository/ocis` folder of your moodle installation
+        - run `composer install` inside of the `repository/ocis` folder
+2. Install [oCIS](https://doc.owncloud.com/ocis/next/quickguide/quickguide.html)
+   - NOTE: the TLS certificates of oCIS need to be trusted by the server running moodle, so if you are using self-signed certificates you need to copy them to the moodle server and make it trust them. e.g. on Debian based systems to run oCIS on `https://host.docker.internal:9200`:
+     1. create a TLS certificate
+        ```bash
+        openssl req -x509  -newkey rsa:2048 -keyout ocis.pem -out ocis.crt -nodes -days 365 -subj '/CN=host.docker.internal'
+        ```
+     2. make 'host.docker.internal' resolve to the IP 127.0.0.1 on the docker host machine
+        ```bash
+        sudo sh -c "echo '127.0.0.1 host.docker.internal' >> /etc/hosts"
+        ```
+     3. run oCIS using this certificate: 
+        ```bash
+        OCIS_INSECURE=true \
+        PROXY_HTTP_ADDR=0.0.0.0:9200 \
+        OCIS_URL=https://host.docker.internal:9200 \
+        PROXY_TRANSPORT_TLS_KEY=./ocis.pem \
+        PROXY_TRANSPORT_TLS_CERT=./ocis.crt \
+        ./ocis server
+        ```
+        :exclamation: Having set `OCIS_INSECURE=true` is not recommended for production use! :exclamation:
+     4. copy the certificate file to the store of the oCIS server and make the system trust it
+        - if oCIS runs on the same server as moodle:
+          ```bash
+          sudo cp ocis.crt /usr/local/share/ca-certificates
+          sudo update-ca-certificates
+          ```
+        - if moodle runs in the development docker container from point 1:
+          ```bash
+          docker cp ocis.crt moodle-docker-webserver-1:/usr/local/share/ca-certificates/
+          docker exec  moodle-docker-webserver-1 update-ca-certificates
+          ```
 3. Login to moodle as "admin"
-4. If you run ocis on `localhost` or any local IP address go to the "HTTP security" page ("Site administration" > "General" > "Security" > "HTTP security") and delete the IP address and host-name you are using from the "cURL blocked hosts list" list
-5. If you run ocis on any port other than `443` go to the "HTTP security" page ("Site administration" > "General" > "Security" > "HTTP security") and add the port you are using to the "cURL allowed ports list" list
+4. If you run oCIS on `localhost` or any local IP address go to the "HTTP security" page ("Site administration" > "General" > "Security" > "HTTP security") and delete the IP address and host-name you are using from the "cURL blocked hosts list" list
```

Comment:
While using local custom host like `host.docker.internal` (in this docs), only this ip `172.16.0.0/12` blocks the oauth2 setup. So I think this should be mentioned at least in the docs.

Reply:
Isn't that what it says: `delete the IP address and host-name you are using`. No need to delete the complete list, only the IP you are using

Reply:
Yes, but just like the example of the service URL was given as `https://host.docker.internal:9200` in `step 4`, at least for the current docker setup, we can include this host as an example of the host that is to be removed. 
Will the IP of `host.docker.internal (172.16.0.0/12)` be different for another system?

Reply:
`host.docker.internal` should refer to `127.0.0.1`
I've updated the docs accordingly

Reply:
While running Moodle from container , this ip `127.0.0.1` doesn't block the service but `172.16.0.0/12` seems to be blocking.